__tests__/app.test.js

```diff
@@ -20,8 +26,436 @@ describe('pokemon routes', () => {
     });
   });
 
+  it('gets a pokemon by id with GET', async () => {
+    const cleffa = await Pokemon.insert({ 
+      name: 'Cleffa',
+      type1: 'Fairy',
+      type2: 'N/A',
+      hiddenAb: 'Friend Guard',
+    });
+
+    const res = await request(app).get(`/api/v1/pokemon/${cleffa.id}`);
+
+    expect(res.body).toEqual(cleffa);
+  });
+
+  it('gets all pokemon with GET', async () => {
+    const cleffa = await Pokemon.insert({ 
```

Comment:
I did want to mention, to make tests a little bit drier, you can add a beforeAll in a test, and create a pokemon. So that it would be, before all tests, a pokemon is inserted. That way, you don't have to insert one for each test. Not necessary for this, but thought it could be a nice suggestion for the future.